extension/src/context.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import * as nls from "vscode-nls";
+import * as Resources from "./resources";
+import { Disposable } from 'vscode-languageclient';
+import { DocumentManager } from './documents';
+
+
+export class ContextManager{
+	private _ctx: vscode.ExtensionContext;
+	private _docManager: DocumentManager;
+	
+	localize = nls.config({ messageFormat: nls.MessageFormat.file })();	
```

Comment:
I put this here because I read that this on https://github.com/Microsoft/vscode-nls
_The config call configures the nls module and should only be called once in the applications entry point._

It seems like the next PR may need to be a general rollout of the code base making use of the context manager. At least for DocumentSelectors & NLS calls anyway.

Reply:
So, what I am suggesting based on that comment from the primary vscode-nls project, is that ALL of our code should be calling the localize() from AutoLispExt.localize() and that the next PR is specifically to apply that minor change across our entire code base.

IE, we are running nls.config() within every single *.TS that needs it, but apparently, we aren't supposed to be using vscode-nls in that way.

Reply:
Thanks for explaining!